src/services/auth.service.js

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+
+const API_URL = 'http://localhost:8080/api/api/auth/';
+
+class AuthService {
+    login(user) {
+        return axios
+            .post(API_URL + 'signin', {
+                username: user.username,
+                password: user.password
+            })
+            .then(response => {
+                if (response.data.accessToken) {
+                    localStorage.setItem('user', JSON.stringify(response.data));
+                }
+
+                return response.data;
+            });
+    }
+
+    logout() {
+        localStorage.removeItem('user');
+    }
+
+    register(user) {
+        return axios.post(API_URL + 'signup', {
+            username: user.username,
+            email: user.email,
+            password: user.password
+        });
+    }
+}
+
+export default new AuthService();
```

Comment:
make this service an object and move to http subdirectory
```suggestion
export default AuthService;
```